pkg/apis/monitoring/v1/types.go

```diff
@@ -489,6 +489,13 @@ type PrometheusSpec struct {
 	// Note that if TargetLimit is lower, that value will be taken instead,
 	// except if either value is zero, in which case the non-zero value will be
 	// used.  If both values are zero, no limit is enforced.
```

Comment:
Nit: Its unrelated to your change but above comment seems to be removed as looks to be that of `EnforcedTargetLimit` which is defined at different place

Reply:
should I remove this comment?

Reply:
Not needed in this PR :)